checker/tests/nullness/RequiresNonNullTest.java

```diff
@@ -143,4 +143,10 @@ void notHiddenTest() {
     void test() {
         o = null;
     }
+
+    // Test case for Issue 1051
+    // https://github.com/typetools/checker-framework/issues/1051
+    //:: error: (flowexpr.parse.error)
```

Comment:
Does this test actually fail if there are two errors? Doesn't it need to be a jtreg test to look at all reported errors?